WooCommerce/WooCommerceTests/ViewRelated/Products/Add Product/AddProductWithAI/ProductCreationAIStartingInfoViewModelTests.swift

```diff
@@ -0,0 +1,98 @@
+import XCTest
+@testable import WooCommerce
+
+final class ProductCreationAIStartingInfoViewModelTests: XCTestCase {
+    private let siteID: Int64 = 123
+    private let sampleImage = MediaPickerImage(image: .addImage, source: .media(media: .fake()))
+
+    func test_imageState_empty_initially() {
+        // Given
+        let sut = ProductCreationAIStartingInfoViewModel(siteID: siteID)
+
+        // Then
+        XCTAssertEqual(sut.imageState, .empty)
+    }
+
+    func test_it_shows_media_picker_source_sheet_upon_tapping_read_text_button() {
+        // Given
+        let sut = ProductCreationAIStartingInfoViewModel(siteID: siteID)
+
+        // When
+        sut.didTapReadTextFromPhoto()
+
+        // Then
+        XCTAssertTrue(sut.isShowingMediaPickerSourceSheet)
+    }
+
+    func test_it_shows_media_picker_source_sheet_upon_tapping_replace_photo_button() {
+        // Given
+        let sut = ProductCreationAIStartingInfoViewModel(siteID: siteID)
+
+        // When
+        sut.didTapReadTextFromPhoto()
```

Comment:
Do you mean `didTapReplacePhoto`?

Reply:
Good catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/13213/commits/d2f11ad39d7f29f368de86b9a74be734cd4ef3b5